packages/gatsby-plugin-feed/src/gatsby-node.js

```diff
@@ -47,12 +34,7 @@ exports.onPostBuild = async ({ graphql }, pluginOptions) => {
       ...feed,
     }
 
-    const serializer =
-      feed.serialize && typeof feed.serialize === `function`
-        ? feed.serialize
-        : serialize
-
-    const rssFeed = (await serializer(locals)).reduce((merged, item) => {
+    const rssFeed = (await feed.serialize(locals)).reduce((merged, item) => {
```

Comment:
Do we want to throw an error if this isn't a function?

Reply:
Won't they get an error anyway? So it might be good to catch this early.